app/src/main/java/com/example/employeeappointmentmaker/AppointmentAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.employeeappointmentmaker
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.employeeappointmentmaker.models.Appointment
+
+class AppointmentAdapter(private var appointments: List<Appointment>) :
+    RecyclerView.Adapter<AppointmentAdapter.AppointmentViewHolder>() {
+
+    inner class AppointmentViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        val titleTextView: TextView = itemView.findViewById(R.id.textViewTitle)
```

Comment:
As seen in [7a91bc1](https://github.com/priyank1968/CloverEmployeeAppointment/pull/19/commits/7a91bc105cdc8e5076febd0d63dec3df4ab56dd6), switched the item getting to viewbinding rather than R.id referencing.